tests/test_adjoint.py

```diff
@@ -42,6 +44,30 @@ class TestAdjoint(object):
         # 3D TTI tests with varying space orders
         ('layers-tti', (30, 35, 40), 'centered', 8, tti_setup),
         ('layers-tti', (30, 35, 40), 'centered', 4, tti_setup),
+        # 2D SLS Viscoacoustic tests with varying space orders
```

Comment:
@mloubout @rhodrin now that I note this... that's a lot of tests added! Do you think we actually need all of them?